src/harness/harness.ts

```diff
@@ -1642,7 +1647,8 @@ module Harness {
         }
 
         function writeComparison(expected: string, actual: string, relativeFilename: string, actualFilename: string, descriptionForDescribe: string) {
-            if (expected != actual) {
+            var encoded_actual = (new Buffer(actual)).toString('utf8')

```

Comment:
Browserify provides it, but it uses typed arrays. You might need to `require("buffer")`.